USca/USca_RTU/MainWindow.xaml.cs

```diff
@@ -51,7 +57,14 @@ private async void SendData()
                 Thread.Sleep(250);
 
                 await CommService.SendSignalsBatch(_reader.Signals);
+                FetchOutputTagValues();
```

Comment:
Can these two api calls not be concurrent? Also, adding `FetchOutputTagValues` here changes the purpose of the function it's in, so the name `SendData` is no longer accurate.

Reply:
[36134cf](https://github.com/magley/untitled-scada-project/pull/21/commits/36134cf7a1a86361fe62fe6d33f938224b7a79fa)